src/System.Numerics.Tensors/System/Numerics/ArrayUtilities.cs

```diff
@@ -69,12 +69,17 @@ public static bool IsDescending(int[] values)
             return true;
         }
 
+        public static int[] GetStrides(int[] dimensions, bool reverseStride = false)
```

Comment:
You know there is an implicit cast from int[] to ReadOnlySpan<int>?

Reply:
Funny enough, it was calling `GetStrides(Array array)` rather than `GetStrides(ReadOnlySpan<int> dimensions, bool reverseStride = false)` when passing in an int[] (previously it would call the latter when it had an int[] in the signature), I renamed the array overload so I can delete this one now.

Reply:
Why do we even have the Array taking method?

Reply:
Yeah, dead code, its been deleted.